internal-registry/eclipse/openshift-cloud-terminal/4.5.0/meta.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+publisher: eclipse
+name: openshift-cloud-terminal
```

Comment:
it's proposed by Mario as well as alternative variant: `eclipse/cloud-shell/openshift4.5`

Reply:
Cloud-terminal is better than cloud-shell. I'm even fine with "openshift-terminal". Thanks!

Reply:
If you're going to rebrand it I have two thoughts here:
a) do you have permission to call it OpenShift ___ ? You need Brand/Marketing/PM approval to use the name in anything producty/projecty like this.

b) if you DO have approval to use OpenShift, then why not use the name that's currently in plan for the productized version?  https://issues.redhat.com/browse/CRW-600 has "OpenShift Command Line Terminal"

cc: @l0rd

Reply:
@l0rd any preferences here?

Reply:
I see that @pdaverh *DID* suggest cloud-terminal is better... but I'm not sure Branding will be cool with "openshift-terminal" or "openshift-cloud-terminal". Has anyone confirmed this naming scheme with the OpenShift PM yet?

Reply:
@sleshchenko that was an error on my side. @nickboldt is right. 

Upstream we should not prefix it with openshift and it would be better if we don't use the eclipse org because that's not an eclipse project yet. `quay.io/che-incubator/command-line-terminal:4.5.0`.

Downstream the product name should be "OpenShift Command Line Terminal" hence we should have `publisher: redhat`, `name: openshift-command-line-terminal`, `version: 4.5.0`. For the image name I don't know what's the best org: `quay.io/redhat/openshift-command-line-terminal:4.5.0`, `quay.io/openshift/command-line-terminal:4.5.0` or `quay.io/crw/openshift-command-line-terminal:4.5.0`?

Reply:
@l0rd and me had a meeting where we agreed on 
`che-incubator/command-line-terminal/4.5.0` for cheEditor used by OpenShift Console for workspaces.
The image that will be in that editor will be tagged as `quay.io/che-incubator/command-line-terminal:4.5.0` but technically it'll be https://github.com/eclipse/che-machine-exec

the PR is updated in accordance, please review and give more feedback if any